markdown/org/docs/designs/shelly/options/straightsides/en.md

```diff
@@ -0,0 +1,10 @@
+---
+title: "Straight sides"
+---
+
+Forms a tubular shirt, which is a shirt where the hips and chest are the same width and connected by a straight seam. Set this to true for a bodice that is perfectly rectangular below the armholes. This setting overrides any values set for the `hips ease` or `side shape` options. For most body types, leaving this option on will result in a reasonable fit that is looser around the belly and tighter around the chest, and generally produces a more fabric-efficient pattern.
```

Comment:
```suggestion
Forms a tubular shirt, which is a shirt where the hips and chest are the same width and connected by a straight seam. Set this to true for front and back parts that are perfectly rectangular below the armholes. This setting overrides any values set for the `hips ease` or `side shape` options. For most body types, leaving this option on will result in a reasonable fit that is looser around the belly and tighter around the chest, and generally produces a more fabric-efficient pattern.
```
Again refered to as body in code, however refering to body here implies person so use front and back instead.